Makefile

```
@@ -21,6 +21,16 @@
 export APPDIR = $(CURDIR)
 include $(APPDIR)/Make.defs
```

```suggestion
include $(CURDIR)/Make.defs
```
and move line 21 as else statement at line 30

---

done

---

sorry, ci failed, we move line 21 as else statement at line 30, because ../apps/Make.defs will use `APPDIR`